staging/src/k8s.io/kubelet/pkg/apis/stats/v1alpha1/types.go

```diff
@@ -269,6 +269,17 @@ type VolumeStats struct {
 	// Reference to the PVC, if one exists
 	// +optional
 	PVCRef *PVCReference `json:"pvcRef,omitempty"`
+
+	// VolumeHealthStats contains data about volume health
+	// +optional
+	VolumeHealthStats *VolumeHealthStats `json:"volumeHealthStats,omitempty"`
+}
+
+// VolumeHealthStats contains data about volume health.
+type VolumeHealthStats struct {
```

Comment:
Since we introduce Message filed in volume.Metrics. If there is any reason that VolumeHealthStats not includes a Message filed ?

Reply:
I am ok to add it. I implement this PR based on the [latest KEP](https://github.com/kubernetes/enhancements/pull/2900/files?short_path=15ee612#diff-15ee61212fda1ec3f574c71df1ad7c03847a33089b9d68740aa68bc598797822)

Reply:
I just feel, for a metric, it should be self-explanation, message is unnecessary

Reply:
Maybe @xing-yang could explain this?

Reply:
Yes, message is for events. We are not using message to generate events here so a message field won't be used.